README.adoc

```diff
@@ -24,6 +24,11 @@ toc::[]
 [[img-build-status]]
 image::https://ci.centos.org/buildStatus/icon?job=vagrant-service-manager[link="https://ci.centos.org/job/vagrant-service-manager"]
 
+== Prerequisites
```

Comment:
There is already a link to ADB in the top section, but maybe we can find yet another way to link back.